charts/brigade-project/templates/secret.yaml

```diff
@@ -28,6 +28,10 @@ stringData:
   {{- else }}
   vcsSidecar: {{ printf "deis/git-sidecar:v%s" .Chart.Version }}
   {{- end }}
+  vcsSidecarResources.limits.cpu: {{ default "" .Values.vcsSidecarResources.limits.cpu | quote }}
```

Comment:
I believe this section needs to be wrapped in a `  {{ if .Values.vcsSidecarResources }}...{{- end}}` block to satisfy the chart linter (`make test-chart` currently throws an error)